discovery-infra/tests/base_test.py

```diff
@@ -63,15 +63,17 @@ def get_cluster_func(cluster_name: Optional[str] = None,
                              additional_ntp_source: Optional[str] = consts.DEFAULT_ADDITIONAL_NTP_SOURCE,
                              openshift_version: Optional[str] = env_variables['openshift_version'],
                              user_managed_networking=False,
-                             high_availability_mode=consts.HighAvailabilityMode.FULL):
+                             high_availability_mode=consts.HighAvailabilityMode.FULL,
+                             olm_operators=env_variables['olm_operators']):
```

Comment:
```suggestion
                             olm_operators=env_variables.get('olm_operators', None):
```

Reply:
We have default value for this variable set [here](https://github.com/openshift/assisted-test-infra/pull/601/files#diff-fd319f80628e389daf226c94bb6aa50ae1fce37b80a4b7221f6da81fbe0094f8R65)